TechItEasy/src/main/java/nl/novi/TechItEasy/Controllers/TelevisionsController.java

```diff
@@ -0,0 +1,69 @@
+package nl.novi.TechItEasy.Controllers;
+
+import nl.novi.TechItEasy.Exceptions.IndexOutOfBoundsException;
+import nl.novi.TechItEasy.Exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
```

Comment:
Je kunt hier nog @RequestMapping("/televisions") toevoegen dan kun je "/televisions" weg halen uit alle andere endpoints die je aanmaakt in deze controller